react/features/notifications/reducer.ts

```diff
@@ -90,6 +90,12 @@ ReducerRegistry.register<INotificationsState>('features/notifications',
  * queue.
  */
 function _insertNotificationByPriority(notifications: INotification[], notification: INotification) {
+
+    // We need to filter out the notifications from queue that have the same id as the new notifications,
```

Comment:
I think before filtering we should get the index, because if the new notification has a given UUID we want it to replace the existing notification, in the exact same position.

Reply:
Will test this.